packages/terra-button/src/Button.scss

```diff
@@ -54,74 +50,86 @@
       padding: 0;
```

Comment:
So firefox automatically adds padding without this set?

Reply:
Yeah, Firefox gives focus to the inner elements and will apply two focus rings. This is a screenshot with these styles removed
![screen shot 2017-12-15 at 3 30 18 pm](https://user-images.githubusercontent.com/14099737/34061247-f7dea8e0-e1ac-11e7-930f-45a8966a57e7.png)

Reply:
Yeah we need to keep these styles to normalize button styles in Firefox.